packages/terra-functional-testing/src/services/wdio-visual-regression-service/methods/BaseCompare.js

```diff
@@ -1,27 +1,128 @@
 /* eslint-disable class-methods-use-this, no-unused-vars */
+import path from 'path';
+
+
+const TEST_ID_REGEX = /\[([^)]+)\]/;
 
 export default class BaseCompare {
+  constructor(options) {
+    const {
+      baseScreenshotDir,
+      locale,
+      theme,
+    } = options;
+
+    // screenshot naming config
+    this.baseScreenshotDir = baseScreenshotDir || process.cwd();
+    this.locale = locale || 'en';
+    this.theme = theme || 'terra-default-theme';
+  }
+
   /**
    * You can do here your image comparison magic.
+   * @param {Object} context - information provided to process the screenshot
+   * @param {Object} context.browser - { name, version, userAgent }
+   * @param {Object} context.suite - the test suite that is running
+   * @param {Object} context.test - the test that is running
+   * @param {Object} context.meta - { element, exclude, hide, remove, viewport}
+   * @param {*} base64Screenshot - the screenshot captured by the selenium command to process.
    */
-  async processScreenshot(context, base64Screenshot) {
+  async processScreenshot(_context, _base64Screenshot) {
     return Promise.resolve();
   }
 
+  /**
+   * Creates the sanitized test name for the screenshot.
+   * @param {String} fullName - test name
+   * returns {String} test name
+   */
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
This appears to be disabled for the entire file (first line of the file). Is this necessary or is the file level disable not working?

Reply:
Originally I had implemented this in the LocalCompare class but move it to the BaseCompare class so the SaveScreenshot compare method can follow the same naming format. I'll remove.